consumer.go

```diff
@@ -111,14 +111,25 @@ func (c *Consumer) Poll(timeoutMS int) (*rdkafka.Message, error) {
 	case rdkafka.OffsetsCommitted:
 		c.log.Print(e)
 	case rdkafka.Error:
-		return nil, errors.New(e.String())
+		if isErrorTerminal(e) {
+			return nil, errors.New(e.String())
+		}
+
+		c.log.Printf("Consumer: Poll: Transient error: %s , code: %d", e.String(), e.Code())
 	default:
 		c.log.Printf("Unknown event type: %T", e)
 	}
 
 	return nil, nil
 }
 
+// isErrorTerminal is a utility function checking if the returned errors should be treated as transient.
+// It will probably get replaced with a librdKafka provided function, informing on each error's severity
+// https://github.com/edenhill/librdkafka/issues/1987#issuecomment-422008750
+func isErrorTerminal(err rdkafka.Error) bool {
```

Comment:
How about something simpler like `IgnorableErr()`? This way we keep a consistent wording (in commit, documentation & code).

The logic in the body will be easier to read too.

Reply:
Actually, I think the function is not needed. If we ever reuse the same piece of code in more spots, then we could convert it to a function. Just a comment above the relevant piece of code with the link to edenhill's answer should be sufficient.